src/sentry/plugins/providers/integration_repository.py

```diff
@@ -82,18 +82,34 @@ def create_repository(
     ):
         result = self.build_repository_config(organization=organization, data=repo_config)
 
+        integration_id = result.get("integration_id")
+        external_id = result.get("external_id")
+
         repo_update_params = {
             "external_id": result.get("external_id"),
```

Comment:
nit: use new variable
```suggestion
            "external_id": external_id,
```